hydra/controllers/network.py

```diff
@@ -315,3 +344,114 @@ def configure(self):
             pubkey = networks[name]['node_data'][ip]['pubkey']
             self.app.network.run_command(ip, f'cd {name}; ./shipchain call register_candidateV2 {pubkey} '
                                          f'{fee} {lock_time} --name shipchain-node-{index + 1} -k node_priv.key')
+
+    @ex(
+        help='pull_registry',
```

Comment:
Can you add a little helptext blurb about what this does? It's not immediately obvious imo